src/components/AdjustPosition/AdjustPosition.tsx

```diff
@@ -683,6 +719,11 @@ const AdjustPosition: FC<AdjustPositionProps> = ({ position }) => {
     return formatPercentage(selectedCollateralBorrowRate);
   }, [borrowingFeeAmount, selectedCollateralBorrowRate]);
 
+  const interestRateFormatted = useMemo(() => {
```

Comment:
This is not merge correctly, we have a code for loading interest rate from contracts.

Reply:
this is the code from `feature/bridge` and `master`? cannot find the code you mention